dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ExtendedMetricsTRSOpenApiIT.java

```diff
@@ -86,7 +86,6 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
-
```

Comment:
Heh, remember our merge issue yesterday? :) But there's a ticket on this so we can deal with it later.